app.js

```diff
@@ -7,19 +7,49 @@ const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const app = express();
 const punkAPI = new PunkAPIWrapper();
 
+// punkAPI.getRandom()
+//   .then(beersFromApi => console.log('Beers from the database: ', beersFromApi))
+//   .catch(err => console.log(err)); 
+
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
 
-app.use(express.static(path.join(__dirname, 'public')));
-
-// Register the location for handlebars partials here:
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
-// ...
+app.use(express.static(path.join(__dirname, 'public')));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers', { beersFromApi });
+    })
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beers', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
+      res.render('random-beer', { responseFromAPI });
```

Comment:
everything looks good on your handlers! another way within your routes, in your res.render, you can make an object variable that you can call in your hbs files; this is mostly for either customising the name you or your team can work best with, or to save your response for a more structured and controlled process.